tsconfig.tsup.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "composite": false,
+    "jsx": "react",
+    "types": ["@cloudflare/workers-types", "node", "ws"]
```

Comment:
`tsup` struggles to understand composite projects, so it's simpler to just disable them

It also needs a reference to any types needed during build, though the `jsx` configuration can probably just go in `tsconfig.base.json` 🤔